### DISCUSSION TITLE
Availability of Helmfile in Homebrew

### DISCUSSION THREAD

**Discussion by philmtd** [helmfile/helmfile#190] [General]
Hello! Now that the first release in the new repository is done (🥳), will this release be available on Homebrew soon?

I guess there may be some extra work to get Homebrew to use the releases from this repository and that's why it's not available yet, right?

**Comment by muffl0n** [comment]
https://github.com/Homebrew/homebrew-core/pull/104878

**Comment by muffl0n** [comment]
Formula is merged and should be available via update/install.
